code/modules/antagonists/cult/cult_bastard_sword.dm

```diff
@@ -0,0 +1,95 @@
+
+/// Cult Bastard Sword, earned by cultists when they manage to sacrifice a heretic.
+/obj/item/cult_bastard
+	name = "bloody bastard sword"
+	desc = "An enormous sword used by Nar'Sien cultists to rapidly harvest the souls of non-believers."
+	w_class = WEIGHT_CLASS_HUGE
+	block_chance = 50
+	throwforce = 20
+	force = 35
+	armour_penetration = 45
+	throw_speed = 1
+	throw_range = 3
+	sharpness = SHARP_EDGED
+	light_color = "#ff0000"
+	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "rends")
+	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "rend")
+	icon = 'icons/obj/cult/items_and_weapons.dmi'
+	icon_state = "cultbastard"
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	lefthand_file = 'icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/64x64_righthand.dmi'
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	actions_types = list()
+	item_flags = SLOWS_WHILE_IN_HAND
+	///are we using our right click ability yet, mr krabs?
```

Comment:
```suggestion
	///are ya spinning, son?
```

Reply:
its an auto documentation not a meme place

Reply:
i'm gonna do neither how about that